docs/install_guides/installing-openvino-yocto.md

```diff
@@ -1,40 +1,86 @@
 # Create a Yocto Image with Intel® Distribution of OpenVINO™ toolkit {#openvino_docs_install_guides_installing_openvino_yocto}
+
 This document provides instructions for creating a Yocto image with Intel® Distribution of OpenVINO™ toolkit.
 
 ## System Requirements
+
+@sphinxdirective
+.. tab:: Supported Linux Distributions
+
+   +--------------------------------+----------------------------------------+
+   | Operating System               | Version                                |
+   +================================+========================================+
+   | Ubuntu                         | 18.04 (LTS), 20.04 (LTS), 22.04 (LTS)  |
+   +--------------------------------+----------------------------------------+
+   | Fedora                         | 34, 35                                 |
+   +--------------------------------+----------------------------------------+
+   | AlmaLinux                      | 8.5                                    |
+   +--------------------------------+----------------------------------------+
+   | Debian GNU/Linux               | 10.x (Buster), 11.x (Bullseye)         |
+   +--------------------------------+----------------------------------------+
+   | OpenSUSE Leap                  | 15.3                                   |
+   +--------------------------------+----------------------------------------+
+
+.. tab::  Required Software
+
+   +---------------------------------------------+---------------------------+
+   | Software                                    | Version                   |
+   +=============================================+===========================+
+   | `GIT <https://git-scm.com/>`__              | 1.8.3.1 or greater        |
+   +---------------------------------------------+---------------------------+
+   | `tar <https://www.gnu.org/software/tar/>`__ | 1.28 or greater           |
+   +---------------------------------------------+---------------------------+
+   | `Python <https://www.python.org/>`__        | 3.6 or greater            |
```

Comment:
Probably 3.6 is dropped for 2022.3 (but not for 2022.1 and 2022.2). @anujm1 could you help confirm?

Reply:
These look like requirements on build machine for a Yocto build to succeed so not related to what is required for OpenVINO.

But in general, I think we should point to Yocto documentation here instead of copying content from there. These keep changing and it'd be difficult to keep this in sync.

Reply:
Will point to Yocto docs, lest we have this info out of sync.